app/templates/orders/view.hbs

```diff
@@ -24,12 +24,12 @@
     </div>
   </div>
   <div class="row">
-    <div class="column right aligned">
-      <button class="ui labeled icon {{if isLoadingTickets 'loading'}} button" {{action downloadTickets model.event.name model.order.identifier}}>
+    <div class="column right {{if device.isMobile 'center'}} aligned">
```

Comment:
Not much difference is created with this.

Reply:
@mohitm15  Try with `fluid` class once.

Reply:
Okay